src/pool/srv_pool.c

```diff
@@ -1112,6 +1322,19 @@ pool_svc_step_up_cb(struct ds_rsvc *rsvc)
 		goto out;
 	event_cb_registered = true;
 
+	/* Update pool map by IV */
+	rc = pool_map_update(svc, map_version, map_buf);
+	if (rc) {
+		D_ERROR("pool_map_update failed "DF_RC"\n", DP_RC(rc));
+		D_GOTO(out, rc);
+	}
+
+	rc = ds_pool_iv_prop_update(svc->ps_pool, prop);
```

Comment:
Is this also a sync IV_SYNC? Do we really have to do sync bcasts (via IV) within in step_up_cb? I thought the idea of using IVs is that we do not have to; other servers will find and ask the leader for IV values.

Reply:
TBH, just for playing safe to avoid race condition. Probably turns to async once we pass the current chaos time?